azure-sql/migration-guides/modernization.md

```diff
@@ -28,7 +28,7 @@ A [migration to Azure](https://azure.microsoft.com/overview/cloud-migration-bene
 
 #### What is Azure SQL?
 
-[Azure SQL](../azure-sql-iaas-vs-paas-what-is-overview.md) is a family of services that use the SQL Server database engine in the Azure Cloud. The following services belong to Azure SQL: [Azure SQL Database](../database/sql-database-paas-overview.md) (SQL Database), [Azure SQL Managed Instance](../managed-instance/sql-managed-instance-paas-overview.md) (SQL Managed Instance) and [SQL Server on Azure VMs](../virtual-machines/index.yml).
+[Azure SQL](../azure-sql-iaas-vs-paas-what-is-overview.md) is a family of services that use the SQL Server database engine in the Azure Cloud. The following services belong to Azure SQL: [Azure SQL Database](../database/sql-database-paas-overview.md) (SQL Database), [Azure SQL Managed Instance](../managed-instance/sql-managed-instance-paas-overview.md) (SQL Managed Instance) and [SQL Server on Azure VMs](../virtual-machines/index.yml) (SQL VM).
```

Comment:
In a lot of cases, we aren't allowed to abbreviate service names. This is one such example.

```suggestion
[Azure SQL](../azure-sql-iaas-vs-paas-what-is-overview.md) is a family of services that use the SQL Server database engine in the Azure Cloud. The following services belong to Azure SQL: [Azure SQL Database](../database/sql-database-paas-overview.md) (SQL Database), [Azure SQL Managed Instance](../managed-instance/sql-managed-instance-paas-overview.md) (SQL Managed Instance) and [SQL Server on Azure VMs](../virtual-machines/index.yml).
```

Reply:
But you are using SQL VM later in the same article. few times like in lines 45, 59, 75, 77, 91, 105, 149, 155, 159, and none of those abbreviations have been written by me. I just see it logical to add this shortened form at the beginning of the article for better understanding.

Reply:
They shouldn't be there then, so I'll have to remove all references to that. That's part of the excitement of inheriting content.

Reply:
Happy to help then :)